vendor/templates/vendor/includes/cost_overview.html

```diff
@@ -32,9 +32,11 @@
     <span class="text-primary">${{ invoice.total|default_if_none:"0.00"|floatformat:2 }}</span>
   </li>
 
-  {% if invoice.get_next_billing_date %}
+  {% if invoice.get_billing_dates_and_prices.items %}
```

Comment:
technically you shouldn't have to do an if check before your for loop here! if theres no items it should just skip this section